Library/Homebrew/caveats.rb

```diff
@@ -94,7 +94,7 @@ def keg_only_text(skip_reason: false)
         end
```

Comment:
If neither of these conditionals are met, we don't get the newline added. Should we add something like this to make sure it's consistently done?

```ruby
if !(formula.lib/"pkgconfig").directory? && !(formula.share/"pkgconfig").directory?
  s << "\n"
end
```

(there's also a spot above that's similar)

Reply:
I did this a different way, but yes.